proto/zitadel/session/v2alpha/challenge.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package zitadel.session.v2alpha;
+
+import "google/protobuf/struct.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+option go_package = "github.com/zitadel/zitadel/pkg/grpc/session/v2alpha;session";
+
+enum ChallengeResponseKind {
+    CHALLENGE_RESPONSE_KIND_UNSPECIFIED = 0;
+    CHALLENGE_RESPONSE_KIND_PASSKEY = 1;
+}
+
+message Challenges {
+  message Passkey {
+    google.protobuf.Struct public_key_credential_request_options = 1 [
```

Comment:
I totally agree on that but the user has to understand our code and the one of the lib, that's where I'm currently struggling at. 
I've added a task to my list to understand the protocol. For the moment i would at least add a link to the spec, so that the user understands what's happening and change it later if required. therefore I'm happy as I see the link to the spec somewhere